examples/typescript/transfer_coin.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ * Similar to ./simple_transfer.ts, but uses transferCoinTransaction to generate the transaction.
+ */
+
+import { Account, AccountAddress, Aptos, AptosConfig, Network } from "aptos";
+
+const COIN_STORE = `0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>`;
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 1_000_000;
+const TRANSFER_AMOUNT = 1_000_000;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ */
+const balance = async (aptos: Aptos, name: string, address: AccountAddress) => {
+  let resource = await aptos.getAccountResource({ accountAddress: address.toUint8Array(), resourceType: COIN_STORE });
```

Comment:
I start to think we want to use `getAccountCoinsData` query instead of this.. imo is cleaner

Reply:
hmm I am actually fine with both haha. Can update if needed :)